build/testdependencies.props

```diff
@@ -0,0 +1,7 @@
+<Project>
+    <Import Project="dependencies.props" />
+    <PropertyGroup>
+        <MicrosoftNETTestSDKPackageVersion>15.0.0</MicrosoftNETTestSDKPackageVersion>
+        <MSTestPackageVersion>1.1.17</MSTestPackageVersion>
```

Comment:
We should avoid sharing package version properties if the packages could rev independently.